test/functional/test_site/testAntiFOUCStyles.md

```diff
@@ -0,0 +1,20 @@
+<!-- Anti-FOUC styles should be applied to navbar. -->
+
+<navbar placement="top" type="inverse">
+  <li>One</li>
+  <li>Two</li>
+</navbar>
```

Comment:
Do you need to test nested dropdowns as well? Either in the `<navbar>` or standalone within the page.

Also the text was hard to spot for the `<li>` within `<navbar>`. Maybe use a light themed one or change `<li>` to use white font colour?

Reply:
Thanks for pointing it out! Wasn't aware that we used the test site for more than HTML output comparison 😅